extensions/INTEL/SPV_INTEL_fpga_latency_control.asciidoc

```diff
@@ -0,0 +1,158 @@
+
+_Latency Label_ is a 32-bit signed integer type scalar that labels the labeled instruction so that it may be referred to in *LatencyControlConstraintINTEL* decorations.
+3+| Literal Number +
+_Latency Label_ | *FPGALatencyControlINTEL*
+
+// --- ROW BREAK ---
+| 6173 | *LatencyControlConstraintINTEL* +
+Apply to an object of type OpTypePointer or OpTypePipe. If that object is used as the pointer operand by an OpLoad or OpStore instruction (for OpTypePointer) or the pipe operand by an OpReadPipe, OpWritePipe, OpReadPipeBlockingINTEL, or OpWritePipeBlockingINTEL instruction (for OpTypePipe) then this decoration conveys latency control information about that instruction. Any such instructions will be referred to as the "constrained instructions" corresponding to the decoration.
+
+_Relative To_, _Control Type_, and _Relative Cycle_ constrain the cycle on which the constrained instruction can be scheduled.
+
+_Relative To_ is a 32-bit signed integer type scalar that identifies the labeled instruction relative to which the constrained instruction associated with this decoration is being constrained. It corresponds to the _Latency Label_ operand of a *LatencyControlLabelINTEL* decoration.
+
+_Relative Cycle_ is a 32-bit signed integer type scalar whose meaning depends on _Control Type_.
+
+_Control Type_ is a 32-bit signed integer type scalar that represents the type of the constraint.
+
+If _Control Type_ is equal to 1, it indicates that the latency between the labeled instruction and the constrained instruction should be exactly _Relative Cycle_ cycles.
+
+If _Control Type_ is equal to 2, it indicates that the latency between the labeled instruction and the constrained instruction should be at most _Relative Cycle_ cycles.
+
+If _Control Type_ is equal to 3, it indicates that the latency between the labeled instruction and the constrained instruction should be at least _Relative Cycle_ cycles.
+| Literal Number +
+_Relative To_ | Literal Number +
+_Control Type_ | Literal Number +
+_Relative Cycle_ | *FPGALatencyControlINTEL*
+|====
+--
+
+*Note* that both of these decorations are ignored for target devices that are not FPGA.
+
+=== Capability
+
+Modify Section 3.31, Capability, adding a row to the Capability table:
+--
+[options="header"]
+|====
+2+^| Capability ^| Implicitly Declares
+| 6171 | FPGALatencyControlINTEL |
+|====
+--
+
+=== Validation Rules
+
+* *Note* that it is invalid for two labeled instructions to be decorated with *LatencyControlLabelINTEL* using the same _Latency Label_.
```

Comment:
Mentioned in last week's SPIR-V call: this condition is already described generally in the main SPIR-V specification so it does not need to be duplicated here.  From [3.20 Decoration](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#_decoration):

> It is invalid to decorate any given id or structure member more than one time with the same decoration, unless explicitly allowed below for a specific decoration.

It would be best to remove this validation rule to eliminate the duplication.